packages/playground/website/src/lib/hooks.ts

```diff
@@ -36,10 +37,23 @@ export function usePlayground({ blueprint, storage }: UsePlaygroundOptions) {
 			remoteUrl.searchParams.set('storage', storage);
 		}
 
+		const onRunningBlueprintSteps = (progressMap: PromiseMap) => {
+			for (const [step, promise] of progressMap.entries()) {
```

Comment:
this is a good place to look at how the naming of the values affects the readability of the intent in the code

Reply:
Refactored as `onBlueprintStepProgress(promise, step)`.